collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/IntentGroupTest.java

```diff
@@ -181,7 +181,7 @@ public void collect_shouldNotCrashWhenAnyExceptionIsThrownWhileReceivingAnswer()
         Intent resultIntent = new Intent();
 
         Uri uri = mock(Uri.class);
-        doThrow(new SecurityException()).when(uri);
```

Comment:
Yeah it looks like this setup is wrong right? `when` is just being passed the `Uri` object. I think it should have been:

```
doThrow(new SecurityException()).when(uri).getScheme();
```

Does that seem right? If so the problem makes sense.

Reply:
Ohh, I see. That makes sense. Seems we should generally go for `when` first anyway, so the fix is good.